src/components/services-view/services-view.html

```diff
@@ -52,6 +52,7 @@
     <div ng-if="$ctrl.currentFilter === 'other' || $ctrl.currentFilter === 'all'" class="services-items">
       <div class="services-items-inner">
         <div class="services-items-inner-shadow-covers">
+          <div ng-if="$ctrl.isEmpty">There are no Catalog Items</div>
```

Comment:
Let's add punctuation and remove the capitalization. We don't capitalize kinds in other parts of the web console.

"There are no catalog items."